cmd/erasure-utils_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"errors"
 	"io"
 	"testing"
+
+	"github.com/aead/siphash"
```

Comment:
Me too, but I think Adam (agl) does not want to include SipHash (yet). I think the reason for this is that it is currently not widely used and the arguments about not including to match stuff to the /x repos - See [17244](https://github.com/golang/go/issues/17244)